.circleci/config.yml

```diff
@@ -21,21 +21,15 @@ jobs:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-exacttarget/bin/activate
-            pylint tap_exacttarget -d no-else-return,wrong-import-order,pointless-string-statement,invalid-name,deprecated-method,bad-whitespace,line-too-long
+            pylint tap_exacttarget -d no-else-return,wrong-import-order,pointless-string-statement,invalid-name,deprecated-method,bad-whitespace,line-too-long,consider-using-sys-exit
```

Comment:
Why do we need to update the pylint?

Reply:
Added a NOTE about that in the PR description.